README.md

```diff
@@ -61,6 +61,81 @@ error about `setuptools-rust` not being found you should upgrade pip with
 `pip install -U pip` or manually install `setuptools-rust` with
 `pip install setuptools-rust` and try again.
 
+### Platform Support
+
+Retworkx strives to support as many platforms as possible, but due to
+limitations in available testing resources and platform availability, not all
+platforms can be supported. Platform support for retworkx is broken into 4
+tiers with different levels of support for each tier. For platforms outside
+these, retworkx is probably still installable, but it’s not tested and you will
+need a Rust compiler and have to build retworkx (and likely Numpy too) from
+source.
+
+Additionally, retworkx only supports CPython. Running with other Python
+interpreters isn’t currently supported.
+
+#### Tier 1
+
+Tier 1 supported platforms are fully tested upstream as part of the development
+processes to ensure any proposed change will function correctly. Pre-compiled
+binaries are built, tested, and published to PyPI as part of the release
+process. These platforms are expected to be installable with just a functioning
+Python environment.
+
+Tier 1 platforms are currently:
+
+- Linux x86_64 (distributions compatible with the manylinux 2010 packaging
+  specification)
+- macOS x86_64 (10.9 or newer)
+- Windows 64 bit
+
+
+#### Tier 2
+
+Tier 2 platforms are not tested upstream as part of development process.
+However, pre-compiled binaries are built, tested, and published to PyPI as part
+of the release process and these packages can be expected to be installed with
+just a functioning Python environment.
+
+Tier 2 platforms are currently:
+
+- Linux i686 (distributions compatible with the manylinux 2010 packaging
+  specification) for Python < 3.10
+- Windows 32 bit for Python < 3.10
+- Linux aarch64 (distributions compatible with the manylinux 2014 packaging
+  specification)
+
+#### Tier 3
+
+Tier 3 platforms are not tested upstream as part of the development process.
+Pre-compiled binaries are built, tested and published to PyPI as
+part of the release process. However, they may not installable with just a
+functioning Python environment and you may be required to build Numpy from
+source, which requires a C/C++ compiler, as part of the installation process.
+
+Tier 3 platforms are currently:
+
+- Linux i686 (distributions compatible with the manylinux 2010 packaging
+  specification) for Python >= 3.10
+- Windows 32bit for Python >= 3.10
```

Comment:
Strictly I think you _only_ build for Python 3.10, so the "> 3.10" part isn't entirely true.  Not sure how precise you care about being, though.

Reply:
I left it this way because I wanted it to be future proof, we're not going to get numpy support for i686 back with 3.11. We might not support 3.11 yet in retworkx but we will at some point and I don't feel like updating this table just to add the `>` in the PR we add 3.11 support.